src/core/batch/batch.ts

```diff
@@ -51,6 +51,21 @@ const parseAnnouncement = <T extends SignedAnnouncement>(record: ParquetRecord):
   return announcement as unknown as T;
 };
 
+const encodeParquetRecord = <T extends AnnouncementType>(announcement: AnnouncementWithSignature<T>): ParquetRecord => {
```

Comment:
This makes sure the values going into Parquet are correct for parquetjs.

Aka most of it is supported out of the box, but hex as byte arrays is not. So this translates those hex values into Uint8Arrays which are supported. `parseAnnouncement` goes the other way back out.